integration-tests/package.mill

```diff
@@ -16,6 +16,14 @@ trait IntegrationTests extends CrossSbtModule with HasScala2Plugin with Scalafmt
   def pluginModule = plugin.cross()
   def millSourcePath = super.millSourcePath / os.up
 
+  override def scalacOptions = Task {
+    if (v.isScala3(crossScalaVersion)) {
+      Seq.empty[String]
+    } else {
+      v.scala2CommonOptions
+    }
+  }
```

Comment:
This function appears three times. Is there a common utility that this should be factored into?

Reply:
Probably--the tricky part is it is a Task (because it needs to create `crossScalaVersion` which is a Task), so I think the only real way to reuse here is with a base trait that we mix in. This PR is just rolling out a pattern we're already using elsewhere, but I agree we should revisit the pattern.

Reply:
Okay I've factored stuff into `HasCommonOptions`.